src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -85,6 +87,7 @@ const RESULTS = l10n.t("Results");
 const MESSAGES = l10n.t("Messages");
 const TIMESTAMP = l10n.t("Timestamp");
 const MESSAGE = l10n.t("Message");
+const QUERY_PLAN = l10n.t("Query Plan");
```

Comment:
Maybe use the locFile in `reactviews/common/locConstants.ts`

Reply:
@caohai and @laurennat, FYI, this approach won’t work because the webview initially loads with default strings. It then requests the extension to retrieve the current language file and updates the strings accordingly. Since these are constant variables, they cannot be updated dynamically after initialization.

Reply:
Thanks @aasimkhan30 ! I created a PR fixing this https://github.com/microsoft/vscode-mssql/pull/18136